### DISCUSSION TITLE
Project Goals

### DISCUSSION THREAD

**Discussion by BenjamenMeyer** [TLNightOps/NightRunner#1] [General]
Setting the goals for what we want to accomplish

**Comment by BenjamenMeyer** [comment]
User Interface for stations

**Reply by evanwieren** [reply]
Yes, with a check in / check out feature.
Would be nice to know when they are done at a station

**Comment by BenjamenMeyer** [comment]
User interface for scoring team

**Reply by evanwieren** [reply]
Bah, who needs good scoring. "Who do we want to win?"  <sarcasm>

**Comment by BenjamenMeyer** [comment]
Reporting

**Comment by BenjamenMeyer** [comment]
Backend API to support

**Reply by evanwieren** [reply]
I think a self contained app is best. No need for front end and backend to be separate. Too much overhead for 2 devs.

**Reply by BenjamenMeyer** [reply]
@evanwieren I'd have to learn a fair bit of front end dev for that most likely...my JS is not up to snuff

**Reply by evanwieren** [reply]
Right. So with DJango or rails, our basis would be covered.

**Comment by BenjamenMeyer** [comment]
Deployment Capability

**Reply by BenjamenMeyer** [reply]
The entire suite needs to be easy to deploy and setup in a secure manner for NightOps organizers as they may or may not have a technical background.

**Reply by BenjamenMeyer** [reply]
We should be resource aware to help minimize costs.

**Reply by evanwieren** [reply]
This is the question. Is it controlled by the central source, or is it something that is allowed for each group to roll out
